### ISSUE TITLE
React example on landing page uses class instead of className

### ISSUE THREAD

**Issue by thetrompf** [linkedin/css-blocks#116]
On the landing page for CSS Blocks in the template.jsx example, you use `class=` instead of `className=` which isn't a valid React prop.

**Comment by amiller-gh** [comment]
You are correct! Left over from our Preact days (where it is valid). Will update 👍

**Comment by mattgreenfield** [comment]
@thetrompf looks like this has been fixed, ok to close?
